lib/Rozier/src/Resources/views/node-types-decorators/actionsMenu.html.twig

```diff
@@ -0,0 +1,12 @@
+{% apply spaceless %}
+    <nav data-action-menu class="actions-menu uk-navbar">
+        {% if (is_granted('ROLE_ACCESS_NODETYPESDECORATORS_DELETE')) %}
```

Comment:
Do not create a new `ROLE_ACCESS_NODETYPESDECORATORS_DELETE` role. Just use the  existing ROLE_ACCESS_NODETYPES